files/initramfs-tools/union-mount.j2

```diff
@@ -11,17 +11,38 @@ case $1 in
     ;;
 esac
 
+docker_inram=false
+logs_inram=false
+secureboot=false
+bootloader=generic
+
+# Extract kernel parameters
+set -- $(cat /proc/cmdline)
+for x in "$@"; do
```

Comment:
The above 2 lines are equal to 'for x in $(cat /proc/cmdline); do', is there a particular reason to use set -- instead?

Reply:
I remember that there are some cases where using `set` has upsides compared to your more straightforward approach.
However it doesn't apply in this case so I'll change it to your suggestion.